e2e/tests/node_reboot.robot

```diff
@@ -129,26 +155,64 @@ Power Off All Worker Nodes For More Than Pod Eviction Timeout While Workload Hea
     END
 
 Reboot Volume Node While Workload Heavy Writing
-    Given Create statefulset 0 with ${VOLUME_TYPE} volume
+    Given Create statefulset 0 using ${VOLUME_TYPE} volume
     FOR    ${i}    IN RANGE    ${LOOP_COUNT}
-        And Keep writing data to statefulset 0
+        And Keep writing data to pod of statefulset 0
 
         When Reboot volume node of statefulset 0
         And Wait for volume of statefulset 0 healthy
-        And Wait for statefulset 0 stable
+        And Wait for statefulset 0 pods stable
 
         Then Check statefulset 0 works
     END
 
 Power Off Volume Node For More Than Pod Eviction Timeout While Workload Heavy Writing
```

Comment:
This test case failed with message:
```
KeyError: 'e2e-test-statefulset-0-0'
```
Could you help check it? Thank you!
https://ci.longhorn.io/job/private/job/longhorn-e2e-test/375/robot/Tests/Node%20Reboot/Power%20Off%20Volume%20Node%20For%20More%20Than%20Pod%20Eviction%20Timeout%20While%20Workload%20Heavy%20Writing/

Reply:
Thanks, should be fixed now.